### PR TITLE
Updated docs website and notebooks

### PR DESCRIPTION
**Pull request by omri374**
## Change Description

Updated and fixed some issues in the docs website:
1. Python APIs using `mkdocstrings` is now rendered correctly
2. Replaced the jupyter package to `mknotebooks` for better rendering and support of opening in Binder
3. Rearranged samples index
4. Each notebook now has a link to its source + pip install cell for execution in binder or Colab.

### CODE REVIEW

**Review by SharonHart — DISMISSED** [review]

**Review by SharonHart — DISMISSED** [review]

File: docs/samples/Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (SharonHart) [inline docs/samples/Pipfile:1]: Commoted by mistake?

**Comment by SharonHart** [comment]
.

File: docs/samples/Pipfile (hunk 1 above)
Comment (omri374) [reply docs/samples/Pipfile:1]: yes.. good catch

Comment (omri374) [reply docs/samples/Pipfile]: @SharonHart removed

**Review by SharonHart — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Pull request merged**